neo.UnitTests/SmartContract/UT_InteropService.cs

```diff
@@ -85,7 +85,7 @@ public void Runtime_GetNotifications_Test()
 
                 // Receive all notifications
 
-                script.EmitPush(UInt160.Zero.ToArray());
```

Comment:
That's true! But one strange thing happened Shargon, shouldn't it FAULT then?

...

Reply:
Oh no! It didn't fault, because 0000...00000 is a valid scripthash, just without any notifications.... makes sense now.